tiup/tiup-troubleshooting-guide.md

```diff
@@ -0,0 +1,56 @@
+---
+title: TiUP Troubleshooting Guide
+category: tools
+---
+
+# TiUP Troubleshooting Guide
+
+This document introduces some common issues when you use TiUP and the troubleshooting methods. If this document does not include the issues you bump into, [file a new issue](https://github.com/pingcap/tiup/issues) in the Github TiUP repository.
+
+## Troubleshoot TiUP commands
+
+### Can't see the latest component list using `tiup list`
+
+TiUP does not update the latest component list from the mirror server every time. You can force refresh the component list by running `tiup list`.
+
+### Can't see the latest version information of a component using `tiup list <component>`
+
+Same as the previous issue, the component version information is only obtained from the mirror server when there is no local cache. You can refresh the component list by running `tiup list <component>`.
+
+### Component downloading process is interrupted
+
+Unstable network might result in an interrupted component downloading process. You can try to download again. If you cannot download it after trying multiple times, it might be caused by the CDN server and you can report the issue [here](https://github.com/pingcap/tiup/issues).
+
+### A checksum error occurs during component downloading process
+
+Because the CDN server has a short cache time, the new checksum file might not match the component package. Try to download again after 5 minutes. If it still does not match, report the issue [here](https://github.com/pingcap/tiup/issues).
+
+## Troubleshoot TiUP cluster component
+
+### Prompt `unable to authenticate, attempted methods [none publickey]` during deployment
+
+During deployment, component packages are uploaded to the remote host and the initialization is performed. This process requires connecting to the remote host. This error is caused by the failure to find the SSH private key to connect to the remote host. 
+
+To solve this issue, confirm whether you specify the private key by running `tiup cluster deploy -i identity_file`:
+
+1. If the `-i` flag is not specified, it might be that TiUP does not automatically find the private key path. It is recommended to explicitly specify the private key path using `-i`.
+2. If the `-i` flag is specified, it might be that TiUP cannot log in to the remote host using the specified private key. You can verify it by manually executing the `ssh -i identity_file user@remote` command.
```

Comment:
```suggestion
+ If the `-i` flag is specified, it might be that TiUP cannot log in to the remote host using the specified private key. You can verify the key by manually executing the `ssh -i identity_file user@remote` command.
```